In [2]:
%load_ext autoreload
%autoreload 2

import numpy as np
import pandas as pd
import datetime
#graphing
import matplotlib.pyplot as plt

In [3]:
entry2_data = pd.read_csv("entry_loc2.csv")

## Table 1 - Aradillas-Lopez (2011)

In [4]:
# Outcomes observed
total_obs = 951
no_firm_entered = 263
hd = 487
only_hd = 124
low = 564
only_lo = 201
both_entered = 363

In [5]:
# No firm entered
no_firm_entered / total_obs * 100

27.655099894847528

In [6]:
# Home Depot entered
hd / total_obs * 100

51.209253417455315

In [7]:
# Only HD entered
only_hd / total_obs * 100

13.038906414300735

In [8]:
# Lowe's entered
low / total_obs * 100

59.305993690851736

In [11]:
# Only Lowe's entered
only_lo / total_obs * 100

21.13564668769716

In [13]:
# Both firms entered
both_entered / total_obs * 100

38.170347003154575

## CDPs

### Total number of stores

In [16]:
total_stores = len(entry2_data)
total_stores

17216

In [17]:
# Total Lowe's
total_low = len(entry2_data[entry2_data['LOW']>0])
total_low

1192

In [19]:
total_low / total_stores * 100

6.9237918215613385

In [18]:
# Total HD
total_hd = len(entry2_data[entry2_data['HD']>0])
total_hd

1255

In [20]:
total_hd / total_stores * 100

7.289730483271375

There are total 1192 Lowe's and 1255 Home Depot stores. There are more HD stores.

### Single entry

In [21]:
# Lowe's with no HD presence
only_low = len(entry2_data[(entry2_data['LOW']>0) & (entry2_data['HD']==0)])
only_low

548

In [23]:
only_low / total_stores * 100

3.183085501858736

In [22]:
# HD with no Lowe's presence
only_HD = len(entry2_data[(entry2_data['LOW']==0) & (entry2_data['HD']>0)])
only_HD

611

In [24]:
only_HD / total_stores * 100

3.549024163568773

Home Depot enters more markets with no Lowe's presence.

### None Enter

In [27]:
# Markets with no stores
none = len(entry2_data[(entry2_data['LOW']==0) & (entry2_data['HD']==0)])
none

15413

In [28]:
none / total_stores * 100

89.52718401486989

### Both Enter

In [8]:
# Number of markets Lowe's enters where HD has a presence
len(entry2_data[(entry2_data['LOW']>0) & (entry2_data['HD']>0)]) \
    - len(entry2_data[(entry2_data['LOW']>0) & (entry2_data['HD']==0)])

96

In [25]:
both = len(entry2_data[(entry2_data['LOW']>0) & (entry2_data['HD']>0)])
both

644

In [26]:
both / total_stores * 100

3.7407063197026025

There are 644 common markets for both Lowe's and Home Depot. 

### Demographics

In [10]:
# Median income for Lowe's 
np.average(entry2_data[entry2_data['LOW']>0]['income_per_capita'])

29190.489932885906

In [11]:
# Median income for HD's markets
np.average(entry2_data[entry2_data['HD']>0]['income_per_capita'])

31595.29322709163

Home Depot enters markets with higher median income.

In [12]:
# Average population for Lowe's 
np.average(entry2_data[entry2_data['LOW']>0]['population'])

70177.18456375839

In [13]:
# Average population for HD's 
np.average(entry2_data[entry2_data['HD']>0]['population'])

82425.24143426295

Home Depot enters markets with higher population.

## CBSA

In [14]:
entry2_data_CBSA = entry2_data[entry2_data['population']>10000]

### Total number of stores

In [15]:
# Total Lowe's
len(entry2_data_CBSA[entry2_data_CBSA['LOW']>0])

972

In [16]:
# Total HD's
len(entry2_data_CBSA[entry2_data_CBSA['HD']>0])

1056

There are total 972 Lowe's and 1057 Home Depot stores. There are more HD stores.

### Single entry

In [17]:
len(entry2_data_CBSA[(entry2_data_CBSA['LOW']>0) & (entry2_data_CBSA['HD']==0)])

384

In [18]:
len(entry2_data_CBSA[(entry2_data_CBSA['LOW']==0) & (entry2_data_CBSA['HD']>0)])

468

Home Depot enters more markets with no Lowe's presence.

### Both Enter

In [19]:
# Number of markets Lowe's enters where HD has a presence
len(entry2_data_CBSA[(entry2_data_CBSA['LOW']>0) & (entry2_data_CBSA['HD']>0)]) \
    - len(entry2_data_CBSA[(entry2_data_CBSA['LOW']>0) & (entry2_data_CBSA['HD']==0)])

204

In [20]:
len(entry2_data_CBSA[(entry2_data_CBSA['LOW']>0) & (entry2_data_CBSA['HD']>0)])

588

There are 588 common markets for both Lowe's and Home Depot. 

### Demographics

In [63]:
# Median income for Lowe's 
np.average(entry2_data_CBSA[entry2_data_CBSA['LOW']>0]['income_per_capita'])

29462.10185185185

In [64]:
# Median income for HD's markets
np.average(entry2_data_CBSA[entry2_data_CBSA['HD']>0]['income_per_capita'])

31421.590350047303

Home Depot enters markets with higher median income.

In [65]:
# Average population for Lowe's 
np.average(entry2_data_CBSA[entry2_data_CBSA['LOW']>0]['population'])

84739.7767489712

In [66]:
# Average population for Lowe's 
np.average(entry2_data_CBSA[entry2_data_CBSA['HD']>0]['population'])

97108.2412488174

Home Depot enters markets with higher population.